sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -3475,6 +3713,9 @@ namespace Azure { namespace Storage { namespace Blobs {
         ETag IfNoneMatch;
         Nullable<std::string> IfTags;
         Nullable<std::string> BlobTagsString;
+        Nullable<DateTime> ImmutabilityPolicyExpiry;
+        Nullable<Models::BlobImmutabilityPolicyMode> ImmutabilityPolicyMode;
+        Nullable<bool> LegalHold;
```

Comment:
`HasLegalHold` ?

Reply:
Is this another naming issue in the swagger?

Reply:
This is in `_detail_` namespace. It's not visible to customers anyway. I won't even bother fixing it.